special-cat.md

```diff
@@ -0,0 +1 @@
+A special cat is a sweet and delicious pink squishy cat made out of marshmallows. Its meows are made of candy floss and its coat is spun sugar. If you ask nicely, he will make you a nutella crepe.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.